Source/WebDriver/Session.cpp

```diff
@@ -3211,13 +3212,22 @@ void Session::doLogEntryAdded(RefPtr<JSON::Object>&& message)
     // TODO Get the current stacktrace for assert, error, trace, and warn messages
     // https://bugs.webkit.org/show_bug.cgi?id=282979
 
+    auto messageType = params->getString("type"_s);
+
     auto entry = JSON::Object::create();
-    entry->setString("type"_s, "console"_s);
+    entry->setString("type"_s, messageType);
     entry->setString("level"_s, level);
     entry->setString("text"_s, messageText);
     entry->setValue("timestamp"_s, *timestampValue);
-    entry->setString("method"_s, method);
-    entry->setArray("args"_s, args);
+
+    if (messageType == "console"_s) {
+        entry->setString("method"_s, method);
+        entry->setArray("args"_s, args);
+    } else if (messageType == "javascript"_s) {
+        // Despite not having the stack information, the spec requires this
+        // field for JavaScript messages.
+        entry->setString("stackTrace"_s, emptyString());
```

Comment:
the `ConsoleMessage` has this data.  can we not send it as part of `Automation.logEntryAdded`?

Reply:
Yes, the idea is to do this in a follow-up bug (https://bugs.webkit.org/show_bug.cgi?id=282979), translating it to the format defined by the BiDi spec.